api/v3/kafka-rest-v3-spec.yaml

```diff
@@ -5,10 +5,36 @@ info:
   version: 3.0.0
 
 servers:
-  - url: http://localhost:9391/v3
-    description: Kafka REST Proxy
-  - url: http://localhost:9391/kafka/v3
-    description: Confluent Server
+  - url: "{protocol}://{address}:{port}/v3"
+    description: Kafka REST Proxy (stand-alone). See https://docs.confluent.io/current/kafka-rest/index.html
```

Comment:
Remove the (stand-alone) and (embedded) from description. This is an implementation detail.

Reply:
Done, resolving conversation.